legend-engine-xt-serviceStore-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/connection/authentication/SecuritySchemeLexerGrammar.g4

```diff
@@ -0,0 +1,13 @@
+lexer grammar SecuritySchemeLexerGrammar;
```

Comment:
why these files inside connection? also why is directory name authentication and not securityScheme?

Reply:
fixed